task-1/question-2.js

```diff
@@ -0,0 +1,14 @@
+var rl = require('readline');
+var fs = require('fs');
+var rs = fs.createReadStream('./sample.csv');
+var ws = fs.createWriteStream('output.txt');
```

Comment:
You can probably use `const` instead of `var`

Reply:
done